src/diffpy/pdfgui/gui/extendedplotframe.py

```diff
@@ -32,28 +32,22 @@
 
 DATA_SAVE_ID  = wx.NewId()
 
-
 class ExtendedToolbar(NavToolbar):
     """An extended plotting toolbar with a save and close button."""
 
     def __init__(self, canvas):
+        # override class var to exclude subplots
+        self.toolitems = tuple(el for el in NavToolbar.toolitems\
+                if el[0] != 'Subplots')
         NavToolbar.__init__(self, canvas)
-        # Get rid of the configure subplots button
-        if hasattr(self, 'wx_ids'):
-            self.DeleteTool(self.wx_ids['Subplots'])
-        else:
-            self.DeleteToolByPos(6)
         # Add new buttons
         self.AddSimpleTool(wx.ID_PRINT,
                wx.ArtProvider.GetBitmap(wx.ART_PRINT, wx.ART_TOOLBAR),
                'Print', 'print graph')
         self.AddSimpleTool(DATA_SAVE_ID,
-               _load_bitmap('stock_save_as.xpm'),
+               _load_bitmap('filesave.png'),
```

Comment:
> but the icons/ is actually at the same level as src/ in this repo.

Ugh, I see the same problem.  Seems to happen only in development-mode installation.

I will fix it later today.  In the meantime, create symbolic links `src/icons --> ../icons`, `src/doc --> ../doc` in your working directory, but make sure they are not added to git.